template/studio/schemas/post.js

```diff
@@ -0,0 +1,122 @@
+const {format} = require('date-fns')
+
+export default {
+  type: 'document',
+  name: 'post',
+  title: 'Blog Post',
+  fields: [
+    {
+      name: 'title',
+      type: 'string',
+      title: 'Title',
+      description: 'Titles should be catchy, descriptive, and not too long'
+    },
+    {
+      name: 'slug',
+      type: 'slug',
+      title: 'Slug',
+      description:
+        'Some frontends will require a slug to be set to be able to show the post',
+      options: {
+        source: 'title',
+        maxLength: 96
+      }
+    },
+    {
+      name: 'publishedAt',
+      type: 'datetime',
+      title: 'Published at',
+      description:
+        'This can be used to schedule post for publishing',
+    },
+    {
+      name: 'mainImage',
+      title: 'Main image',
+      type: 'mainImage',
+    },
+    {
+      name: 'excerpt',
+      type: 'text',
```

Comment:
This should probably be a portable text type, to allow for using code and emphasize text (but not links)?

Reply:
The excerpt is used in post listings and summaries, and on Google? Does it makes sense to have links and stuff in that?

Reply:
It's used in the list of posts on the front page as well. Should not have links, but bold, italics, code, etc.